BREAKING.md

```diff
@@ -10,6 +10,12 @@ There are a few steps you can take to write a good change note and avoid needing
 - Provide guidance on how the change should be consumed if applicable, such as by specifying replacement APIs.
 - Consider providing code examples as part of guidance for non-trivial changes.
 
+## 0.56 Breaking Changes
+- [`Save` operation type removed from `SummarizeReason`](#save-operation-type-removed-from-summarizeReason)
+
+### `Save` operation type removed from `SummarizeReason`
+The `Save` operation type was deprecated and has now been removed from `SummarizeReason` in the `container-runtime` package. Additionally, `MessageType.Save` and `MessageFactory.createSave()` have been deprecated from `protocol-definitions` and `server-test-utils` respectively. Both will be removed in a future release.
```

Comment:
I think this text (first part) will confuse people more than it helps, as they will struggle to understand if they should care or not. I'd just say that MessageType.Save and code that handled it was removed. It should be noop if MessageType.Save  was not used by consumers.